src/ripple/app/misc/ValidatorList.h

```diff
@@ -241,6 +241,18 @@ class ValidatorList
         return quorum_;
     }
 
+    /** Get maximum quantity of trusted validators allowed to disagree
+     * while still allowing consensus (converse of quorum).
+     *
+     * @return quantity allowed to disagree.
+     */
+    std::size_t
+    allowedDisagree () const
+    {
+        std::lock_guard<std::shared_timed_mutex> lock{mutex_};
+        return trustedKeys_.size() - quorum_;
```

Comment:
Is there a guarantee that `trustedKeys_.size() >= quorum_`? Otherwise this could return a massive number.

Reply:
i beieve that quorum_ can be 0xFFFFFFFF when lists are not available from calculateQuroum